src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/DataReaderTest/DataReaderTest.cs

```diff
@@ -790,6 +790,95 @@ static char[] Resize(char[] buffer)
+
+            using (var cn = new Microsoft.Data.SqlClient.SqlConnection(DataTestUtility.TCPConnectionString))
+            {
+                await cn.OpenAsync();
+
+                string tableName = DataTestUtility.GenerateObjectName();
+
+                try
+                {
+                    using (var cmd = cn.CreateCommand())
+                    {
+                        cmd.CommandText = $"""
+                   if exists (select *
```

Comment:
@Wraith2 I think you can just remove this "drop" section - and maybe put the table name in [ ] ?

Reply:
fixed.